docs/cfn-service-role.yml

```diff
@@ -0,0 +1,175 @@
+                  - events:RemoveTargets
+                  - glue:CreateDatabase
+                  - glue:CreateTable
+                  - glue:DeleteDatabase
+                  - glue:DeleteTable
+                  - iam:AddRoleToInstanceProfile
+                  - iam:AttachRolePolicy
+                  - iam:CreateInstanceProfile
+                  - iam:CreatePolicy
+                  - iam:CreateRole
+                  - iam:DeleteInstanceProfile
+                  - iam:DeletePolicy
+                  - iam:DeletePolicyVersion
+                  - iam:DeleteRole
+                  - iam:DeleteRolePolicy
+                  - iam:DetachRolePolicy
+                  - iam:GetInstanceProfile
+                  - iam:GetPolicy
+                  - iam:GetRole
+                  - iam:GetRolePolicy
+                  - iam:ListPolicyVersions
+                  - iam:PassRole
+                  - iam:PutRolePolicy
+                  - iam:RemoveRoleFromInstanceProfile
+                  - lambda:AddPermission
+                  - lambda:CreateEventSourceMapping
+                  - lambda:CreateFunction
+                  - lambda:DeleteEventSourceMapping
+                  - lambda:DeleteFunction
+                  - lambda:DeleteLayerVersion
+                  - lambda:GetEventSourceMapping
+                  - lambda:GetFunction
+                  - lambda:GetLayerVersion
+                  - lambda:InvokeFunction
+                  - lambda:PublishLayerVersion
+                  - lambda:PutFunctionConcurrency
+                  - lambda:RemovePermission
+                  - lambda:UpdateFunctionCode
+                  - lambda:UpdateFunctionConfiguration
+                  - logs:CreateLogGroup
+                  - logs:DeleteLogGroup
+                  - logs:DeleteResourcePolicy
+                  - logs:DescribeLogGroups
+                  - logs:DescribeResourcePolicies
+                  - logs:PutResourcePolicy
+                  - logs:PutRetentionPolicy
+                  - rds:AddTagsToResource
+                  - rds:CreateDBInstance
+                  - rds:CreateDBSubnetGroup
+                  - rds:DeleteDBInstance
+                  - rds:DeleteDBSubnetGroup
+                  - rds:DescribeDBInstances
+                  - rds:DescribeDBSubnetGroups
+                  - rds:ModifyDBInstance
+                  - s3:CreateBucket
+                  - s3:DeleteBucket
+                  - s3:DeleteBucketPolicy
+                  - s3:GetBucketPolicy
+                  - s3:GetObject
+                  - s3:PutBucketCORS
+                  - s3:PutBucketPolicy
+                  - s3:PutBucketVersioning
+                  - s3:PutLifecycleConfiguration
+                  - sqs:CreateQueue
+                  - sqs:DeleteQueue
+                  - sqs:GetQueueAttributes
+                  - ssm:DeleteParameter
+                  - ssm:GetParameters
+                  - ssm:PutParameter
+                Resource: '*'
```

Comment:
How can we improve this, tagging?

Reply:
Yes, I think having Quilt in its own account is desirable for various reasons (e.g. service quotas).